cmd/podman/main.go

```diff
@@ -48,6 +48,7 @@ var mainCommands = []*cobra.Command{
 	_pullCommand,
 	_pushCommand,
 	_rmiCommand,
+	_stopCommand,
```

Comment:
there should be a corresponding _stopCommand in commands.go which will need to be removed.  Also, then in containers.go add _StopCommand and in commands.go find getContainerSubCommand and remove stop there too.